test/gen-testfiles

```diff
@@ -18,11 +23,17 @@ mockery -name Interface -print \
     |sed 's/type Interface/type Ringpop/g' \
     > "$mock_directory"/ringpop.go
 
-echo
-
 mockery -name NodeInterface -recursive -print \
     |sed 's/_m \*NodeInterface/_m \*SwimNode/g' \
     |sed 's/type NodeInterface/type SwimNode/g' \
     > "$mock_directory"/swim_node.go
 
 mockery -case=underscore -dir router -recursive -output "$mock_directory" -name ClientFactory
+
+# make a EventListenerMock for forward (can't use on from $mock_directory because of circular dependency)
+mockery -name EventListener -dir events -recursive -print \
+    |sed 's/package mocks/package forward/g' \
+    > "${0%/*}/../forward/mock_event_listener_test.go"
```

Comment:
Sorry, I understood the package adjustment—I didn't follow "${0%/*}" as a bash operation.  What exactly does that one do?